### PR TITLE
Update Travis badge to Github Actions badge

### PR DESCRIPTION
**Pull request by akhilveeraghanta**
### Description
Need to update main badge from travis to github actions

### Testing Done
Verified badge showed up

### Resolved Issues
N/A

### Length Justification
n/a

### Review Checklist

**_It is the reviewers responsibility to also make sure every item here has been covered_**

- [x] **Function & Class comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the functions defined in `thunderbots/software/geom`. Similarly, all classes should have an associated Javadoc comment explaining the purpose of the class.
- [x] **Remove all commented out code**
- [x] **Remove extra print statements**: for example, those just used for testing
- [x] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
- [x] **Justify drops in code coverage**: If this PR results in a non-trivial drop in code coverage (a bot should post a coverage diagram as a comment), please justify why we can't test the code that's not covered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 TODO (https://github.com/UBC-Thunderbots/Software/issues/1478): Uncomment this once Appveyor is up and running again
 [![Build status](https://ci.appveyor.com/api/projects/status/a7106hhbcal8kqi8?svg=true)](https://ci.appveyor.com/project/UBCThunderbots/software)
 -->
-[![Build Status](https://travis-ci.org/UBC-Thunderbots/Software.svg?branch=master)](https://travis-ci.org/github/UBC-Thunderbots/Software/builds)
+![Tbots CI](https://github.com/UBC-Thunderbots/Software/workflows/Tbots%20CI/badge.svg)
```
Comment (jonathanlew) [inline README.md]: clicking on the icon should link to the github actions ci for master

**Review by jonathanlew — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (akhilveeraghanta) [reply README.md]: fixed

**Review by jonathanlew — APPROVED** [review]
LGTM!

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 TODO (https://github.com/UBC-Thunderbots/Software/issues/1478): Uncomment this once Appveyor is up and running again
 [![Build status](https://ci.appveyor.com/api/projects/status/a7106hhbcal8kqi8?svg=true)](https://ci.appveyor.com/project/UBCThunderbots/software)
 -->
-[![Build Status](https://travis-ci.org/UBC-Thunderbots/Software.svg?branch=master)](https://travis-ci.org/github/UBC-Thunderbots/Software/builds)
+[![Tbots CI](https://github.com/UBC-Thunderbots/Software/workflows/Tbots%20CI/badge.svg)](https://github.com/UBC-Thunderbots/Software/actions?query=workflow%3A%22Tbots+CI%22+branch%3Amaster)
```
Comment (BrianS99) [inline README.md:6]: LGTM!  I was wondering what exactly a Github badge does. Is it to improve readability for the CI we are using?

**Review by BrianS99 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (akhilveeraghanta) [reply README.md:6]: its just a badge that is on the main readme when you open the repo to indicate the status of our CI

**Pull request merged**